origami.json

```
@@ -1,5 +1,6 @@
 {
-	"description": "Fetch API helper tool that fallbacks to JSONP",
+	"description": "Client side fetch wrapper that fallbacks to JSONP (to circumvent cross-domain issues involved with sending JSON data)",
```

Maybe move the part in brackets to a separate sentence? This is stylistically closer to your other descriptions
```suggestion
	"description": "Client-side fetch wrapper that fallbacks to JSONP. This circumvents cross-domain issues involved with sending JSON data",
```